sample/src/main/java/com/esafirm/sample/MainActivity.java

```diff
@@ -138,8 +138,8 @@ public void start() {
                 .theme(R.style.ImagePickerTheme)
                 .returnAfterFirst(returnAfterCapture) // set whether pick action or camera action should return immediate result or not. Only works in single mode for image picker
                 .folderMode(folderMode) // set folder mode (false by default)
-                .folderTitle("Folder") // folder selection title
-                .imageTitle("Tap to select"); // image selection title
+                .folderTitle(getString(R.string.ef_title_folder)) // folder selection title
```

Comment:
Hi @JonasHae. I think we don't need to extract this String especially to the library since this is only a sample.